src/jobs/jobs.router.js

```diff
@@ -2,18 +2,85 @@ const winston = require('winston');
 const controller = require('./jobs.controller');
 const slackService = require('../slack/slack.service');
 const teamsController = require('../teams/teams.controller');
-const jobService = require('./jobs.service');
+const jobsUtils = require('./jobs.utils');
+const jobsService = require('./jobs.service');
 const Responses = require('../constants/responses');
 
 /**
- * Create a new job job.
- * If the job already exists in the database, just broadcast to the channel
+ * Handle /job post command
  */
 async function post(req, res) {
   winston.info('jobs-router:post', { payload: req.body });
 
+  const slackMessage = req.body.text;
+  const slackCommand = jobsUtils.getCommand(slackMessage);
+
+  return commandFunctions[slackCommand](req, res);
```

Comment:
A switch statement makes the intention of this method clear

```js
switch (jobsUtils.getCommand(slackMessage)) {
    case COMMANDS.CREATE_JOB:
      return _createJob(req, res);
    case COMMANDS.HELP:
      return _help(req, res);
...
    default:
      break;
```

Reply:
It may be clearer, but this one is faster.